### ISSUE TITLE
[RFE] Fedora 36 rpms / repo

### ISSUE THREAD

**Issue by joe-lawrence** [beaker-project/beaker#155]
The Beaker download page https://beaker-project.org/download.html provides a fedora.repo file, but no packages exist for Fedora 36 (yet):
```
$ sudo dnf install beaker-redhat
Beaker Client - Fedora36                        541  B/s | 237  B     00:00    
Errors during downloading metadata for repository 'beaker-client':
  - Status code: 404 for https://beaker-project.org/yum/client/Fedora36/repodata/repomd.xml (IP: 8.43.85.221)
Error: Failed to download metadata for repo 'beaker-client': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried
```

**Comment by StykMartin** [comment]
Beaker-redhat package is definitely not provided in BP.org repository. However, I synced F36 client there.

**Comment by joe-lawrence** [comment]
@StykMartin : Thanks, Martin and good catch on the package name.  Do you happen to know who to contact to update the internal RH beakerrepos?

**Comment by StykMartin** [comment]
I would start with @p3ck. Bill can help with this for sure.

**Comment by p3ck** [comment]
@cbouchar is building this in dist-git so it should be available soon for f36.

**Comment by cbouchar** [comment]
@joe-lawrence @p3ck build folks fixed their issue and beaker-redhat 0.2.1-4 should be in place as well as -3 if you really need that.
